front/src/features/filter/__tests__/getSelectedTimeFromURL.ts

```diff
@@ -1,27 +1,61 @@
+import { TimeOption } from "@/types/logDisplayOptions";
 import "@testing-library/jest-dom";
 import { getSelectedTimeFromURL } from "../helpers/getSelectedTimeFromURL";
 
 describe("getSelectedTimeFromURL", () => {
-    it("should return 'Custom interval' if startDateTime and endDateTime are defined", () => {
-        const startDateTime = "2021-08-01T00:00:00.000Z";
-        const endDateTime = "2021-08-02T00:00:00.000Z";
-        const dateTimeFilter = undefined;
-        const result = getSelectedTimeFromURL(
-            startDateTime,
-            endDateTime,
-            dateTimeFilter,
+    describe("clean format", () => {
+        test.each([
+            [
+                'should return "Custom interval" if startDateTime and endDateTime are defined',
+                "2021-08-01T00:00:00.000Z",
+                "2021-08-02T00:00:00.000Z",
+                undefined,
+                "Custom interval",
+            ],
+            [
+                "should return 'Last hour' if dateTimeFilter is not defined",
+                undefined,
+                undefined,
+                undefined,
+                "Last hour",
+            ],
+            [
+                "should return 'Last hour' if dateTimeFilter is last-hour",
+                undefined,
+                undefined,
+                "last-hour" as TimeOption,
+                "Last hour",
+            ],
+            [
+                "should return 'Last day' if dateTimeFilter is last-day",
+                undefined,
+                undefined,
+                "last-day" as TimeOption,
+                "Last day",
+            ],
+            [
+                "should return 'Last week' if dateTimeFilter is last-week",
+                undefined,
+                undefined,
+                "last-week" as TimeOption,
+                "Last week",
+            ],
+        ])(
+            "%s",
+            (
```

Comment:
instead of having the test description in the array it might be cleaner to do something like:

`it.each([`
        [arguments, "expected thing"],...
    ])("should return '%s' when dateTimeFilter is '%s'",